src/app/Shared/Services/Api.service.tsx

```diff
@@ -1087,20 +1106,20 @@ export class ApiService {
     );
   }
 
-  groupHasRecording(group: EnvironmentNode, filter: ActiveRecordingFilterInput): Observable<boolean> {
+  groupHasRecording(group: EnvironmentNode, filter: ActiveRecordingsFilterInput): Observable<boolean> {
     return this.graphql<any>(
       `
-    query GetRecordingForGroup ($groupFilter: EnvironmentNodeFilterInput, $recordingFilter: ActiveRecordingFilterInput){
+    query GroupHasRecording ($groupFilter: DiscoveryNodeFilterInput, $recordingFilter: ActiveRecordingsFilterInput){
       environmentNodes(filter: $groupFilter) {
         name
         descendantTargets {
           name
-          recordings {
-              active(filter: $recordingFilter) {
-                  data {
-                    name
-                  }
+          target {
+            activeRecordings(filter: $recordingFilter) {
+              aggregate {
+                count
```

Comment:
should the body stay `data { name }`?

Reply:
No, the adjustments I made a few lines down (https://github.com/cryostatio/cryostat-web/pull/1224/files/08aca67f3d6d6a8897573778b53a2be3f94b9063#diff-310840475f629e447da0a3a6ae5c23f6f3fb856923f21478a25d85073a2c614bR1136) account for this change in the query.